no-computer.md

```diff
@@ -58,12 +104,32 @@ making language design decisions), as opposed to "systems" aspects of the
 project (e.g., implementing a complicated semantics that doesn't require a lot
 of language design)?_
 
+_Once someone got an idea to work on this project, they would have to decide which
+pieces they want to include in their DSL. So, this would all be engaging with the
+language because they would have to think about how the blocks can interact with
+each other, how the colors interact with each other, and how many of each block is
```

Comment:
I like how there's definitely a concept of fluency in this no-computer DSL, with the physical fitting of blocks as well as the visual fluency from color.